mucommander-os-linux/src/main/java/com/mucommander/desktop/linux/kde/KdeTrash.java

```diff
@@ -116,7 +118,7 @@ public boolean canEmpty() {
 
     @Override
     public boolean empty() {
-        return executeAndWait(EMPTY_TRASH_COMMAND);
+        return executeAndWait(trashCommand+" --empty");
```

Comment:
this relies on having the `--empty` argument to future commands as well - how about injecting `<trash_cmd> --empty` instead of only the trash_cmd and providing the `--empty` argument here?

Reply:
Good think. Thanks! Fixed.